connectors/php/filemanager.class.php

```diff
@@ -198,14 +200,7 @@ public function getinfo() {
 		$this->item['properties'] = $this->properties;
 		$this->get_file_info('', false);
 		
-		// handle path when set dynamically with $fm->setFileRoot() method
-		if($this->dynamic_fileroot != '') {
```

Comment:
Sorry for my English. when I tried to upload a file, you gave it to me the full path to the file system